docs/components/light.md

```diff
@@ -50,6 +50,14 @@ The example below creates a light source shining from the upper-left at a 45-deg
 <a-entity light="type: directional; color: #EEE; intensity: 0.5" position="-1 1 0"></a-entity>
 ```
 
+It is also possible to specify the direction of the directional light with it's orientation by creating a child entity it targets. for example pointing down it's -Z axis:
```

Comment:
I think the example below is a bit confusing because it includes `position` + `rotation` + `target` vs. just `position` or just `rotation`. Not sure why we'd want a nested target with `position` when we can just do `position`?